example/yt/compute_soliton_CoM_velocity_by_yt.py

```diff
@@ -0,0 +1,55 @@
+import yt
+import numpy as np
+from scipy import constants
+
+# yt reference
+# 1. gradient fields
+#    https://yt-project.org/doc/analyzing/fields.html?highlight=gradient#gradient-fields 
+#    example:
+#    https://yt-project.org/doc/cookbook/calculating_information.html#complicated-derived-fields
+# 2. weight average
+#    https://yt-project.org/doc/reference/api/yt.data_objects.derived_quantities.html?highlight=center%20mass#yt.data_objects.derived_quantities.WeightedAverageQuantity
+
+def momentum_field_x(field, data):
+   grad_fields_real_x = data["gamer", "Real_gradient_x"]
+   grad_fields_imag_x = data["gamer", "Imag_gradient_x"]
+   momentum_field_x   = ( data["gamer","Real"]*grad_fields_imag_x - data["gamer","Imag"]*grad_fields_real_x )/data["gamer","Dens"]
+   return momentum_field_x
+
+def momentum_field_y(field, data):
+   grad_fields_real_y = data["gamer", "Real_gradient_y"]
+   grad_fields_imag_y = data["gamer", "Imag_gradient_y"]
+   momentum_field_y   = ( data["gamer","Real"]*grad_fields_imag_y - data["gamer","Imag"]*grad_fields_real_y )/data["gamer","Dens"]
+   return momentum_field_y
+
+def momentum_field_z(field, data):
+   grad_fields_real_z = data["gamer", "Real_gradient_z"]
+   grad_fields_imag_z = data["gamer", "Imag_gradient_z"]
+   momentum_field_z   = ( data["gamer","Real"]*grad_fields_imag_z - data["gamer","Imag"]*grad_fields_real_z )/data["gamer","Dens"]
+   return momentum_field_z
```

Comment:
Clarify that there are actually **velocity*m/hbar** in the unit of **1/code_length** instead of momentum.